airbyte-integrations/connectors/source-corebos/source_corebos/libs/WSClient.py

```diff
@@ -0,0 +1,355 @@
+# -*- coding: utf-8 -*-
+'''
+Copyright (c) 2015, David Fernández González, All rights reserved.
+
+This library is free software; you can redistribute it and/or
+modify it under the terms of the GNU Lesser General Public
+License as published by the Free Software Foundation; either
+version 3.0 of the License, or (at your option) any later version.
+
+This library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public
+License along with this library.
+'''
+
+import urllib.request
+import urllib.parse
+import hashlib
+import json
+import re
+
+__author__ = 'David Fernández González'
+__license__ = 'LGPL'
+__version__ = '0.1.2'
+__maintainer__ = 'David Fernández González'
+__email__ = 'dfernandez.parking@gmail.com'
+__status__ = 'Development'
+__credits__ = []
```

Comment:
why not use the lib and install using pip in the setup.py?

Reply:
The lib is not yet hosted and it's custom.